durabletask/worker.py

```diff
@@ -228,8 +228,8 @@ def __init__(self, instance_id: str):
         self._is_replaying = True
         self._is_complete = False
         self._result = None
-        self._pending_actions = dict[int, pb.OrchestratorAction]()
-        self._pending_tasks = dict[int, task.CompletableTask]()
+        self._pending_actions = {}
+        self._pending_tasks = {}
```

Comment:
Let’s add type hints to these variable declarations.

Reply:
Yeah, added type hints whereever required.